frontend/javascripts/oxalis/view/right-border-tabs/bounding_box_tab.js

```diff
@@ -1,30 +1,100 @@
- * @flow
- */
-import { Tooltip } from "antd";
+// @flow
+import { Button, Form, Modal, Tooltip } from "antd";
 import { PlusSquareOutlined } from "@ant-design/icons";
 import { useSelector, useDispatch } from "react-redux";
 import React, { useState } from "react";
 import _ from "lodash";
 
-import { setPositionAction } from "oxalis/model/actions/flycam_actions";
-import type { Vector3, Vector6, BoundingBoxType } from "oxalis/constants";
+import { DatasetNameFormItem } from "admin/dataset/dataset_components";
 import { UserBoundingBoxInput } from "oxalis/view/components/setting_input_views";
-import { getSomeTracing } from "oxalis/model/accessors/tracing_accessor";
+import type { Vector3, Vector6, BoundingBoxType } from "oxalis/constants";
 import {
   changeUserBoundingBoxAction,
   addUserBoundingBoxAction,
   deleteUserBoundingBoxAction,
 } from "oxalis/model/actions/annotation_actions";
+import { getActiveSegmentationTracingLayer } from "oxalis/model/accessors/volumetracing_accessor";
+import { getBaseSegmentationName } from "oxalis/view/right-border-tabs/segments_tab/segments_view_helper";
+import { getSomeTracing } from "oxalis/model/accessors/tracing_accessor";
+import { setPositionAction } from "oxalis/model/actions/flycam_actions";
+import { startGlobalizeFloodfillsJob } from "admin/admin_rest_api";
+import ExportBoundingBoxModal from "oxalis/view/right-border-tabs/export_bounding_box_modal";
+import Toast from "libs/toast";
 import * as Utils from "libs/utils";
+import features from "features";
 
-import ExportBoundingBoxModal from "oxalis/view/right-border-tabs/export_bounding_box_modal";
+const GLOBALIZE_FLOODFILL_REGEX = /Limits of flood-fill \(source_id=(\d+), target_id=(\d+), seed=([\d,]+), timestamp=(\d+)\)/;
+
+function StartGlobalizeFloodfillsModal({
+  onStartGlobalization,
+  handleClose,
+  activeUser,
+  initialName,
+}) {
+  const handleSubmit = async formValues => {
+    try {
+      await onStartGlobalization(formValues.name);
+      handleClose();
+    } catch (exception) {
+      Toast.info(
+        <React.Fragment>
+          The globilization of the floodfill operation(s) could not be started. Open the browser
+          console for more details.
+        </React.Fragment>,
+      );
+      console.error(exception);
+      return;
+    }
+
+    Toast.info(
+      <React.Fragment>
+        The globilization of the floodfill operation(s) has been started. For large datasets, this
+        may take a while. Closing this tab will not stop the computation.
+        <br />
+        See{" "}
+        <a target="_blank" href="/jobs" rel="noopener noreferrer">
+          Processing Jobs
+        </a>{" "}
+        for an overview of running jobs.
+      </React.Fragment>,
+    );
+  };
+
+  return (
+    <Modal title="Globalize Floodfills" onCancel={handleClose} visible width={500} footer={null}>
+      <p>
+        For this annotation some floodfill operations were aborted prematurely, because they covered
```

Comment:
Maybe a more positive framing? :)

```suggestion
        For this annotation some floodfill operations have not run to completion, because they covered
```